make-up/code/5094-viktor-lozanovski/src/app/app.component.html

```diff
@@ -1,4 +1 @@
-<app-header></app-header>
-<router-outlet></router-outlet>
-<app-footer [student]="student$ | async" ></app-footer>
-
+<app-recipe-list></app-recipe-list>
```

Comment:
By removing the `router-outlet` you ensured that the routing does not work as expected.